rust/README.md

```diff
@@ -51,8 +59,24 @@ cargo run --example read_csv
 
 ## Run Tests
 
+Parquet support in Arrow requires data to test against, this data is in a
+git submodule.  To pull down this data run the following:
+
+```bash
+git submodule update --init
+```
+
+The data can then be found in `cpp/submodules/parquet_testing/data`.
+Create a new environment variable called `PARQUET_TEST_DATA` to point
+to this location and then `cargo test` as usual.
+
+Our CI uses `rustfmt` to check code formatting.  Although the project is
+built and tested against nightly rust we use the stable version of
+`rustfmt`.  So before submitting a PR be sure to run the following
+and check for lint issues:
+
 ```bash
-cargo test
+cargo +stable fmt --all -- --check
```

Comment:
@paddyhoran How about [creating a pre-commit](https://pre-commit.com/#new-hooks) hook for this?
We have our pre-commit hooks here: https://github.com/apache/arrow/blob/master/.pre-commit-config.yaml

Reply:
Yep, sounds like a good idea...